src/test/java/tests/EventBannersTest.java

```diff
@@ -59,7 +59,7 @@ public void eventBannersContent() {
         }
 
         goToPreviousPage(driver);
-        homepage.eventBannerThreeClick();
+        homepage.thirdEventBannerGet();
```

Comment:
the word "get" should come first, call methods what the others contributors call it